collections/_blogs/2020-06-25-SMI-conformance-testing-with-meshery.md

```diff
@@ -11,52 +11,52 @@ permalink: /blog/smi-conformance-testing-with-meshery
 
 ### Meshery'ing with Conformance Tests
 
-**[Meshery](/meshery)** is the **service mesh management plane**, it supports all the popular meshes, how to manage them, apply custom or recommended configuration, tests for checking whether meshes are compatible or not, performing performance tests for mesh and lots more. The SMI conformance testing required performance testing capabiliy, load generation (meshery is about to support distributed load generation as well), and other functionality such that these conformance tests can be easily used in pipeline of all the popular service mesh. Mesheryctl does have a perf command that can be used in pipeline of service meshes, I aim at making such capability for SMI conformance too. 
+**[Meshery](/meshery)** is a **service mesh management plane**. It supports all the popular meshes, teaches you how to manage them, assists you in applying custom or recommended configurations, tests for  compatibility, performs performance tests for meshes and a lot more. The SMI conformance testing requires performance testing capabilities, load generation (Meshery is about to support distributed load generation as well), and other functionalities such that these conformance tests can be easily used in the pipelines of all the popular service meshes. mesheryctl does have a perf command that can be used in the pipelines of service meshes. I aim at making such capabilities for SMI conformance as well.
 
-Ok, I hope by this time you know what SMI is, what SMI conformance mean, how meshery's amazing engineering can be utilized for conformance tests, but there might be questions like how these tests were written, how do we validate the traffic, what tools do we use, let's see everything in the bigger picture.
+As you have made your way halfway through the post (thank you for your patience), you should now be aware of SMI, its conformance practices and how Meshery’s incredible engineering can be utilized for conformance tests. We can now tackle the larger questions and hope to see the bigger picture.
 
-### The Bigger Picture
 
-Do you know that almost all of the test cases that we would write in this project would be just raw YAML files? To those doing traditional unit and integration tests, this might sound crazy at first, maybe crazier than the Joker in Batman series, but wait we are sane, and hopefully average engineers😛 (we would like to think so).
+### The Bigger Picture
 
-Presenting an awesome tool for testing all things related to Kubernetes:
+Do you know that almost all of the test cases that we would write in this project would be  raw YAML files? To those doing traditional unit and integration tests, we might sound unhinged at this point. We assure you that we are completely sober and serious (if you don’t count the temporary euphoria from geeky jokes).
 
-#### [kuttl](https://kuttl.dev/) 
+We present to you an incredibly powerful tool for testing all things related to Kubernetes:
```

Comment:
Let's not describe kuttl as an incredibly powerful tool, rather as "a tool for writing tests against Kubernetes operators and controllers."